ocs_ci/utility/templating.py

```diff
@@ -222,3 +222,18 @@ def dump_data_to_json(data, json_file):
     """
     with open(json_file, 'w') as fd:
         json.dump(data, fd)
+
+
+def json_to_dict(json_file):
```

Comment:
I don't see any usage of this function and if this is not heavily used can we just in place where it's needed to use those two lines you have in function? 

```
    with open(json_file, 'r') as fd:
        return json.loads(fd.read())
```